### ISSUE TITLE
Add tests for every primitive

### ISSUE THREAD

**Issue by blefloch** [blefloch/latex-unravel#18]
Start from the list of primiives in `l3names`.  Go through it and write tests for all of them.

**Comment by jfbu** [comment]
Hi Bruno
will this include `\strcmp` from `XeTeX`? The other day I wanted to try out `unravel` on code using `\strcmp` and it complained.
best,
Jean-François

**Comment by blefloch** [comment]
Thanks. I've logged adding XeTeX's `\strcmp` as a separate issue, #23.  This one is more about testing that all primitives that I claim to support are indeed supported.  (I fear that I won't have to work on `unravel` until November, so all this will wait.)

**Comment by jfbu** [comment]
Makes sense. Not being versed in `expl3` I misunderstood your sentence at top of this issue.
